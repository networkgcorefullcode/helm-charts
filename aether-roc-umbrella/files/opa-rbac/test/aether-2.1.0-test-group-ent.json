{
  "groups": [
    "AetherROCAdmin"
  ],
  "target": "starbucks",
  "site": [
    {
      "site_id": "site_1",
      "display_name": "Site 1",
      "small_cell": [
        {
          "small_cell_id": "sc_1_1",
          "display_name": "SC 1_1"
        },
        {
          "small_cell_id": "sc_1_2",
          "display_name": "SC 1_2"
        }
      ]
    },
    {
      "site_id": "site_2",
      "display_name": "Site 2",
      "small_cell": [
        {
          "small_cell_id": "sc_2_1",
          "display_name": "SC 2_1"
        },
        {
          "small_cell_id": "sc_2_2",
          "display_name": "SC 2_2"
        }
      ]
    }
  ],
  "template": [
    {
      "template_id": "template_1",
      "display_name": "Template 1"
    },
    {
      "template_id": "template_1",
      "display_name": "Template 1"
    }
  ]
}
