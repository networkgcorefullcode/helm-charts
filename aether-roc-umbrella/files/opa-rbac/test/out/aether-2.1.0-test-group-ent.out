[
  {
    "application": [],
    "site": [
      [
        {
          "display_name": "Site 1",
          "site_id": "site_1",
          "small_cell": [
            {
              "display_name": "SC 1_1",
              "small_cell_id": "sc_1_1"
            },
            {
              "display_name": "SC 1_2",
              "small_cell_id": "sc_1_2"
            }
          ]
        },
        {
          "display_name": "Site 2",
          "site_id": "site_2",
          "small_cell": [
            {
              "display_name": "SC 2_1",
              "small_cell_id": "sc_2_1"
            },
            {
              "display_name": "SC 2_2",
              "small_cell_id": "sc_2_2"
            }
          ]
        }
      ]
    ],
    "template": [
      [
        {
          "display_name": "Template 1",
          "template_id": "template_1"
        },
        {
          "display_name": "Template 1",
          "template_id": "template_1"
        }
      ]
    ],
    "traffic_class": []
  }
]
